src/components/Dashboard/MainInput/MainInput.jsx

```diff
@@ -54,27 +36,22 @@ export default class MainInput extends React.Component {
     });
   };
 
-  setInputValue = (value) => {
+  setInputValue = (value = '') => {
     const { newPriority } = this.state;
-    const inputDataArr = value.split('');
+    const inputDataArr = value;
```

Comment:
inputDataArr const is not needed any more

Reply:
It was deleted.